### PR TITLE
Code clean up in Job

### PR DESCRIPTION
**Pull request by ckelleyRH**
* Make logger final
* Reduce visibility of public constructor
* Use built in logger formatting
* Remove unnecessary override of run()
* Remove unnecessary Boolean literal
* Combine identical catch clauses
* Rearrange logic in build methods for readability

A separate change will replace the synchronized HashTable instances

### CODE REVIEW

**Review by edewata — APPROVED** [review]
LGTM

**Review by fmarco76 — APPROVED** [review]
LGTM

File: base/server/src/main/java/com/netscape/cms/jobs/Job.java [hunk 1]
```diff
@@ -110,6 +109,7 @@ public boolean isEnabled() {
         try {
             enabled = mConfig.isEnabled();
         } catch (EBaseException e) {
+            // Swallow exceptions?
```
[the remark this answers is not in the document]
Comment (ckelleyRH) [reply base/server/src/main/java/com/netscape/cms/jobs/Job.java:112]: Yeah I try not to alter the control/exception flow when I am wearing my janitor uniform, but I often document that I have not touched things as I go.

**Comment by ckelleyRH** [comment]
Thanks @edewata @fmarco76 !

**Pull request merged**